### PR TITLE
IoT Hub recommendations

### PR DESCRIPTION
**Pull request by ReneHezser**
# Overview/Summary

ARG queries for IoT Hub

## Related Issues/Work Items

- Infra - IoT - IoT Hub - ARG queries Output Standardization for Automation [AB#31163](https://dev.azure.com/CSUSolEng/12ba7e46-a92a-47f6-ad6a-fba0479234a7/_workitems/edit/31163)

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [ ] Ensured PR tests are passing
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by Dost2010** [comment]
Hello ReneHezser,

Can you please change the query and script outputs to only contain the following outputs/columns: https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#azure-resource-graph-arg-queries

**Comment by ReneHezser** [comment]
Sure, no problem. I didn't read the guidance as "name them paramx", but don't know how to rephrase it to be clearer. Maybe it was just me who didn't understand. Sorry for that.

**Comment by oZakari** [comment]
Thanks for the feedback @ReneHezser, I'll update the contribution guide for the column names to make it clearer.

**Review by lachaves — APPROVED** [review]
Checked files, ran hugo server, LGTM.
